### DISCUSSION TITLE
group maintained clones

### DISCUSSION THREAD

**Discussion by leamas** [opencpn-manuals/main#18] [General]
In reply to https://github.com/opencpn-manuals/plugins/issues/66#issuecomment-922554371

> But I don't see calculator_pi there now!! https://github.com/opencpn-manuals
> So where did it go?

It might be that I just removed it. A fork of a plugin maintained outside of the team has a single purpose: to host a  PR. When the PR is accepted, the repo is useless and should be removed. This is probably what has happened here.

However, there might be situations where there is a local clone which needs to github repo again, probably to make a new PR. If so, the simple remedy is to clone the repo (here climatology_pi) again into opencpn-manuals. Then, reset the repo in the clone using `git remote update origin` and `git rebase origin/master`.  

And that's it.

**Comment by leamas** [comment]
@Rasbats  @rgleason ^

**Comment by Rasbats** [comment]
It was me. I deleted it after I found the PR did not work and I edited my calculator repo directly.

**Reply by leamas** [reply]
Ah... it's your own repo, no need to make a PR, you can and should just edit-commit-push when need be.

OTOH, if Rick for some reason (which?) would want to modify this manual, he'll need to make a PR against your repo. For that he needs a clone, be it in opencpn-manuals or as his own private copy in rgleason. No big deal in this case, I would say.

**Comment by rgleason** [comment]
Ok, that explains some things. Thanks.  I "forked" it from Rasbats github page using my opencpn-manuals profile (I was given a choice) and calculator was replicated in the opencpn-manuals/calculator folder.  

I then made "Edit this page" changes to antora.yml and site.yml and and push those changes using the github interface  (no CLI)

The part that puzzles me is that Rasbats says that that PR did not work. 
Answer (I think):  Using the Edit this page button was using the wrong branch gh-???  and I should have been using CLI commands.

The other part that puzzles me is that I did make successful PR's for "manual site antora titles" from the other PI in opencpn-manuals!
